lib/tasks/admin.rake

```diff
@@ -0,0 +1,15 @@
+namespace :admin do
```

Comment:
Is this getting used somewhere?  It's nice to have, but I don't see it getting called.